EventListener/ValidationRequestListener.php

```diff
@@ -88,8 +91,9 @@ public function onKernelRequest(GetResponseEvent $event)
             $dataObject = new \stdClass();
         }
 
-        // Validate data with given schema.
-        $validator = new \JsonSchema\Validator();
+        $schemaStorage = new SchemaStorage();
+        $schemaStorage->addSchema('file://' . $schemaFile, $schema);
```

Comment:
@wachterjohannes I created an issue (https://github.com/sulu/SuluValidationBundle/issues/12) for that and probably take a look at it at the end of next week.